README.md

```diff
@@ -1,6 +1,9 @@
 This repository holds all packages from Freifunk Frankfurt that are meant to be
 used with the gluon firmware.
 
+The [MIT license](https://raw.githubusercontent.com/freifunk-ffm/packages/master/LICENSE) applies unless otherwise stated. 
```

Comment:
Warum keine relative Referenz auf die Datei?

& Ja, ich weiß -> mimimi, ist aber auch hier eher schöner ;)

Reply:
Nenn mir doch bitte erstmal deine Gründe warum Du das gerne möchtest.
"Ist schöner" reicht mir nicht aus um nochmals aktiv zu werden.

Reply:
Die Datei existiert an der stelle im Repo, sollte sie/es umbenannt/verschoben/whatever werden ist der Link in allen forks kaputt. Same für bspw. einen Lizenzwechsel/Änderung: Dann sagt die link Beschreibung möglicherweise was anderes als der Link auf den sie verweist. Wenn man das Projekt offline lesen will landet man auf Github vs auf dem lokal liegenden file?
Ich sehe eher keinen Grund weshalb man GitHub über einen direkten Link verlinken sollte?

Reply:
`[MIT license](LICENSE)` bspw?

Reply:
Kuck an, dass geht ja sogar. Wieder was gelernt 😺